src/agent.ts

```diff
@@ -18,72 +24,52 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const envPath = path.join(__dirname, '../.env.local');
 dotenv.config({ path: envPath });
 
-type SessionId = `${string}-${string}`;
-
-// Why do we need this?
-// We want to be able to retrieve past conversation witht the user given userId and roomId
-// So we concatenate userId and roomId to construct a unique sesion id specific to a room and user
-const constructUserSessionId = (userId: string, roomId: string): SessionId => {
-  return `${userId}-${roomId}`;
-};
-
-const appendMessageToSession = async (
-  sessionId: SessionId,
-  message: string,
-  role: Zep.RoleType,
-) => {
-  await zep.memory.add(sessionId, {
-    messages: [
-      {
-        content: message,
-        roleType: role,
-      },
-    ],
-  });
-};
-
-const getOrCreateZepSession = async (userId: string, roomId: string) => {
-  const sessionId = constructUserSessionId(userId, roomId);
-
-  console.log('Get or add session', sessionId);
-  try {
-    const session = (await zep.memory.getSession(sessionId)) as Promise<
-      Zep.Session & { sessionId: SessionId }
-    >;
-    return session;
-  } catch (error) {
-    if (error instanceof Zep.NotFoundError) {
-      const newSession = (await zep.memory.addSession({
-        sessionId: sessionId,
-        userId: userId,
-      })) as Promise<Zep.Session & { sessionId: SessionId }>;
-      return newSession;
-    } else throw error;
-  }
+type Config = {
+  id: string;
+  name: string;
+  createdAt: string;
+  updatedAt: string;
+  instruction: string;
+  voice: string | null;
+  organisationId: string | null;
 };
 
-const zepRoleToChatRole = (role: Zep.RoleType): llm.ChatRole => {
-  switch (role) {
-    case Zep.RoleType.AssistantRole:
-      return llm.ChatRole.ASSISTANT;
+const loadConfig = (): Promise<Config | null> => {
+  const url = process.env.CONFIG_SOURCE_URL;
```

Comment:
I'd recommend keeping the names all the way through, to avoid confusion

```suggestion
  const CXCORTEX_AGENT_API_URL= process.env.CXCORTEX_AGENT_API_URL;
```

etc